src/Components/SideBar.js

```diff
@@ -30,27 +30,35 @@ function SideBar() {
         <div class="tabs__container border-bs border-radius">
           <ul class="tabs grid">
             <li>
-              <div class="tab px-1 txt-prm py-0-5">
-                <i class="fa-solid fa-house"></i>
-                <span class="hideMd"> Home </span>
+              <div className="tab px-1 txt-prm py-0-5">
+                <NavLink to="/home">
+                  <i class="fa-solid fa-house"></i>
+                  <span class="hideMd"> Home </span>
+                </NavLink>
               </div>
             </li>
             <li>
-              <div class="tab px-1 py-0-5">
-                <i class="fa-regular fa-building"></i>
-                <span class="hideMd"> Explore </span>
+              <div className="tab px-1 py-0-5">
+                <NavLink to="/explore">
+                  <i class="fa-regular fa-building"></i>
+                  <span class="hideMd"> Explore </span>
+                </NavLink>
               </div>
             </li>
             <li>
-              <div class="tab px-1 py-0-5">
-                <i class="fa-regular fa-bookmark"></i>
-                <span class="hideMd"> BookMark </span>
+              <div className="tab px-1 py-0-5">
+                <NavLink to="/bookmark">
+                  <i class="fa-regular fa-bookmark"></i>
+                  <span class="hideMd"> BookMark </span>
+                </NavLink>
               </div>
             </li>
             <li>
-              <div class="tab px-1 py-0-5">
-                <i class="fa-solid fa-user-tie"></i>
-                <span class="hideMd"> Profile </span>
+              <div className="tab px-1 py-0-5">
+                <NavLink to="/profile">
+                  <i class="fa-solid fa-user-tie"></i>
+                  <span class="hideMd"> Profile </span>
+                </NavLink>
               </div>
             </li>
```

Comment:
This looks like a repetative code you can create an array with the name and endpoint, and loop through it